resource-types/CoralogixEvent.dhall

```diff
@@ -0,0 +1,51 @@
+{ meta =
```

Comment:
@ari-becker you need to have support for all the pipelines FYI as it mostly relevant to the service pipelines it should also be supported as part of the old pipeline for now